tests/integration/nupic/opf/hotgym_regression_test.py

```diff
@@ -64,7 +64,8 @@ def testHotgymRegression(self):
       # Changes that affect prediction results will cause this test to fail.
       # If the change is understood and reviewers agree that there has not been a
       # regression then this value can be updated to reflect the new result.
-      self.assertAlmostEqual(float(lastRow[14]), 5.92657292088)
+      # self.assertAlmostEqual(float(lastRow[14]), 5.92657292088)
```

Comment:
ok, thanks. It's hard to tell if it's significant, but it's definitely better. Let's remove the old line before merging in. Thanks!